float64 lat
float64 lon
float64 hei
float32 vel_n
float32 vel_e
float32 vel_g
float32 acc_x
float32 acc_y
float32 acc_z
float32 gravity
float32 roll
float32 pitch
float32 heading
float32 ang_x
float32 ang_y
float32 ang_z

